### ISSUE TITLE
How to use the apko-publish action with an APK image built in a previous step by the melange action?

### ISSUE THREAD

**Issue by kastl-ars** [chainguard-dev/actions#453]
I am building a APK package, that is not (yet) included in Wolfi, in a Github Action. I am using both the setup-melange and melange-build-pkg actions.

After the package build, I would like to use that package in the apko-publish action to create an image using the local repository created by the previous step.

I have not found documentation on how to do that, unfortunately.

I got it working, so if someone wants to do something similar:
https://github.com/kastl-ars/wolfi-apkrane

TL;DR: It would be really great to have a README with each action, describing how to use the action. Especially as melange-build-package is running "on the host" aka the Github Runner (and therefore needs the setup-melange action) while e.g. apko-publish runs inside a container...
